### PR TITLE
PD-532: Update 23.10 API documentation for 23.10-BETA.1 release

### PR DESCRIPTION
**Pull request by Mrt134**
Also reviewed the related tutorial and ui reference content and made a few updates.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/API/_index.md [hunk 1]
```diff
@@ -5,16 +5,12 @@ geekdocCollapseSection: true
 weight: 50
 ---
 
-You can access TrueNAS SCALE API documentation in the web interface by clicking **<i class="material-icons" aria-hidden="true">account_circle</i> > API Keys > DOCS**.
+You can access TrueNAS SCALE API documentation in the web interface by clicking **<i class="material-icons" aria-hidden="true">account_circle</i> > API Keys > API Docs**.
 
-![SCALEapidocs](/images/SCALE/SCALEapidocs.png "API Docs location")
+![SCALEapidocs](/images/SCALE/23.10/SCALEapidocs.png "API Docs location")
 
 Alternatively, append `/api/docs/` to your TrueNAS hostname or IP address in a browser to access the API documentation.
```
Comment (micjohnson777) [inline content/API/_index.md:12]: host name in this instance is two words:
```suggestion
Alternatively, append `/api/docs/` to your TrueNAS host name or IP address in a browser to access the API documentation.
```

File: content/SCALETutorials/TopToolbar/ManagingAPIKeys.md [hunk 2]
```diff
@@ -57,6 +57,6 @@ To delete, select **Confirm** on the delete dialog to activate the **Delete** bu
 
 ## API Key Documentation
 
-Click **API Docs** to access API documentation for your system.
+Click **API Docs** to access API documentation that is built in to the system.
```
Comment (micjohnson777) [inline content/SCALETutorials/TopToolbar/ManagingAPIKeys.md:60]: should this be: 
```suggestion
Click **API Docs** to access API documentation that is built into the system.
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Two small things, and the RESTful link didn't want to load, it kept spinning. Might just be a slow connection issue.

**Comment by Mrt134** [comment]
Double-checked the restful file and verified it is available from the web server. Issue appears to be related to trying to pull it from a localhost:1313 build.

**Pull request merged**